nemo/collections/asr/data/audio_to_text.py

```diff
@@ -1017,8 +1022,13 @@ def __init__(self, tokenizer):
                 else:
                     self.is_aggregate = False
                 self._tokenizer = tokenizer
-
             def __call__(self, *args):
+                if len(args) == 1 and isinstance(args[0], list):
```

Comment:
Same as above

Reply:
Attempted to address to the best of my understanding